src/electron/platform/android/rule-information-provider.ts

```diff
@@ -79,21 +79,21 @@ export class RuleInformationProvider {
         return result;
     }
 
-    private getTouchSizeUnifiedResolution = (ruleResultsData: RuleResultsData): UnifiedResolution => {
+    private getTouchSizeUnifiedResolution = (ruleResultsData: RuleResultsData): UnifiedFormattableResolution => {
```

Comment:
Should this be getTouchSizeUnifiedFormattedResolution instead of getTouchSizeUnifiedResolution?